internal/manager/run.go

```diff
@@ -234,6 +206,106 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic, d
 	return mgr.Start(ctx)
 }
 
+func validateKongRoots(kongRoots []map[string]any, skipCACerts bool) (string, kong.Version, error) {
+	var (
+		errs        []error
+		kongVersion kong.Version
+		dbMode      string
+	)
+
+	uniqs := lo.UniqBy(kongRoots, func(kongRoot map[string]any) string {
```

Comment:
Why are you using `UniqBy` here? How can we end up having the same root twice?

Reply:
The whole point of using `UniqBy` here is to get "unique" values from provided kong  (configuration) roots but we only take a look at dbmode and version (for now, in subsequent PRs in order to support upgrades where different kong versions are under the same service we might relax this requirement here and add code in other places to account for that).